components/HomeScreen/ContinentsList/ContinentsList.jsx

```diff
@@ -0,0 +1,53 @@
+import { Text, View, ActivityIndicator, Pressable } from "react-native";
+import { theme } from "../../../utils/theme";
+
+
+/**
+ * ContinentsList.
+ *
+ * @param {object} props
+ * @param {object} props.data
+ * @param {boolean} props.loading
+ * @param {function} props.refetch
+ */
+function ContinentsList(props) {
+
+  const {
+    data = {},
+    error = false,
+    loading = false,
+    refetch,
+  } = props;
+
+  return(
+    <View
+      style={{
+        flex: 1,
+        gap: theme.spacing(),
+      }}
+    >
+      { loading ? (
```

Comment:
same feedback here as the previous commit. Nest ternaries cause readability issues, I'd consider doing it a different way

Reply:
Done.